src/NzbDrone.Core/Indexers/Definitions/Cardigann/Cardigann.cs

```diff
@@ -132,7 +132,7 @@ private IndexerDefinition GetDefinition(CardigannMetaDefinition definition)
                 {
                     "private" => IndexerPrivacy.Private,
                     "public" => IndexerPrivacy.Public,
-                    _ => IndexerPrivacy.SemiPublic
+                    _ => IndexerPrivacy.SemiPrivate
```

Comment:
YAML will be `type: semi-private`  should we not handle this explictly?

Reply:
A C# switch needs a default. so it's fine as it is